qiita_db/test/test_study.py

```diff
@@ -147,11 +145,11 @@ def setUp(self):
             'metadata_complete': True,
             'reprocess': False,
             'number_samples_promised': 27,
-            'emp_person_id': qdb.study.StudyPerson(2),
+            'emp_person_id': 2,
```

Comment:
Went is this still the id but the rest are not? We should be consistent.
On Feb 27, 2016 14:17, "Jose Navas" <notifications@github.com> wrote:

> In qiita_db/test/test_study.py
> <https://github.com/biocore/qiita/pull/1647#discussion_r54337870>:
>
> > @@ -147,11 +145,11 @@ def setUp(self):
> >              'metadata_complete': True,
> >              'reprocess': False,
> >              'number_samples_promised': 27,
> > -            'emp_person_id': qdb.study.StudyPerson(2),
> > +            'emp_person_id': 2,
>
> That's incorrect - note how it is actually the id, the ones that are no
> longer the id are the principal_investigator and the lab_person
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1647/files#r54337870>.
>